### ISSUE TITLE
Create 'Health check summary' component to be displayed at the bottom of the canvas in storybook

### ISSUE THREAD

**Issue by greenc05** [bbc/simorgh#9502]
**Is your feature request related to a problem? Please describe.**
This Issue should be completed after https://github.com/bbc/psammead/issues/4572
 
Replicate the following issue components in Simorgh https://github.com/bbc/psammead/issues/4572

**Describe the solution you'd like**
Same as https://github.com/bbc/psammead/issues/4572

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Checklist**
- [ ] (BBC contributors only) This issue follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Additional context**

**Comment by ryanmccombe** [comment]
Note Simorgh's Storybook Deployments have been broken since 9th Sep - should ensure that issue is addressed before picking up this: bbc/simorgh-infrastructure#1463
